client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -188,6 +188,12 @@ const BrewRenderer = (props)=>{
 		}));
 	};
 
+	const styleObject = { height: state.height };
```

Comment:
height:100vh works fine here, i tested in another PR. This works too, but is less readable.